acs-integration-tests/src/test/java/com/ge/predix/cloud/foundry/CloudFoundryUtilities.java

```diff
@@ -0,0 +1,33 @@
+package com.ge.predix.cloud.foundry;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public final class CloudFoundryUtilities {
+
+    private CloudFoundryUtilities() {
+        throw new AssertionError();
+    }
+
+    private static final String APP_AND_SERVICE_NAME_PREFIX = System.getProperty("jenkins.build.number");
```

Comment:
Changing it to `@Value` will disallow the variable from being accessed in `static` contexts.